src/providers/Wagmi.tsx

```diff
@@ -8,7 +8,9 @@ import { alchemyProvider } from "wagmi/providers/alchemy";
 const { chains, provider, webSocketProvider } = configureChains(
   [chain.goerli],
   [
-    alchemyProvider({ apiKey: process.env.REACT_APP_ALCHEMY_API_KEY }),
+    alchemyProvider({
+      apiKey: process.env.REACT_APP_ALCHEMY_API_KEY as string
```

Comment:
i think we should do a null check for this env var and throw an error if its not given - the whole app requires it to work - that way we can avoid the `as string` typecast